yb-voyager/src/utils/utils.go

```diff
@@ -290,7 +290,8 @@ func LookupIP(name string) []string {
 
 	ips, err := net.LookupIP(name)
 	if err != nil {
-		ErrExit("Error Resolving name=%s: %v", name, err)
+		log.Infof("Error Resolving name=%s: %v", name, err)
+		return []string{name}
```

Comment:
This is sneaky! Return an empty slice if the name can't be resolved. But do not play with the semantics of a function. In this case, the callers (or at least someone who is reading the code) would expect that the `LookupIP()` is going to return a list of IP addresses. Please, fix this before merging.

Reply:
ok, will fix it before merging

Reply:
done